torch/nn/_qschme.py

```diff
@@ -0,0 +1,22 @@
+from .._jit_internal import weak_script
+
+# NB: Keep this file in sync with enums in c10/core/QScheme.h
+
+
+@weak_script
+def get_enum(qscheme):
+    # type: (str) -> int
+    if qscheme == 'none':
+        ret = 0
+    elif qscheme == 'per_tensor_affine':
+        ret = 1
+    elif qscheme == 'per_channel_affine':
+        ret = 2
+    elif qscheme == 'per_tensor_symmetric':
+        ret = 3
+    elif qscheme == 'per_channel_symmetric':
+        ret = 4
+    else:
+        ret = -1  # TODO: remove once JIT exceptions support control flow
```

Comment:
You probably want to either return -1 or raise an error - right now assigning -1 is useless since we're throwing on the next line and never return -1.

Reply:
I'm not sure exactly what is happening here.

Reply:
this is copied from _reduction.py

Reply:
I'll remove ret = -1, what does the comment mean?

Reply:
I think there is enough support in JIT for it now - we can script code throwing exceptions. Maybe it was written when it was not the case, and the author meant to keep the raise statement commented out.